src/TConsole.cpp

```diff
@@ -1895,8 +1896,9 @@ bool TConsole::setMiniConsoleFontSize(int size)
     return true;
 }
 
-void TConsole::refreshMiniConsole() const
+void TConsole::refreshMiniConsole()
 {
+    mFontFactor = 1.0;
```

Comment:
Hm - from a design perspective this seems wrong - should this be a separate function?

Reply:
Not sure I understand what you are saying - but it seemed, in testing, a good idea to reset the fudge-factor when the main font being used for a `TConsole` is changed.

Reply:
If this function is only called when the font is changed, maybe we should refactor the function and rename it?